Content.Shared/Storage/EntitySystems/DumpableSystem.cs

```diff
@@ -124,27 +125,41 @@ private void StartDoAfter(EntityUid storageUid, EntityUid targetUid, EntityUid u
 
     private void OnDoAfter(EntityUid uid, DumpableComponent component, DumpableDoAfterEvent args)
     {
-        if (args.Handled || args.Cancelled || !TryComp<StorageComponent>(uid, out var storage) || storage.Container.ContainedEntities.Count == 0)
+        if (args.Handled || args.Cancelled)
+            return;
+
+        DumpContents(uid, args.Args.Target, args.Args.User, component);
+    }
+
+    // Refactor to allow dumping that doesn't require a verb
+    [PublicAPI]
+    public void DumpContents(EntityUid uid, EntityUid? target, EntityUid user, DumpableComponent? component = null)
+    {
+        if (!TryComp<StorageComponent>(uid, out var storage)
+            || !Resolve(uid, ref component))
+            return;
+
+        if (storage.Container.ContainedEntities.Count == 0)
```

Comment:
DumpableSystem.cs refractor to allow dumping that doesn't require a verb, which is required for the functionality of Rodentia mouth storage being spilled when damaged.
this also changes lines 149, 155, 158, 162 and 182

Reply:
Don't put line numbers in the comment. Just write one in each method along the lines of "CD: Split into two methods to allow dumping that doesn't require a verb, which is required for the functionality of Rodentia mouth storage being spilled when damaged." Try to avoid words like "refactor" because that does not actually explain what changed, just that something did.